docs/onelake/onelake-onecopy-quickstart.md

```diff
@@ -48,6 +48,7 @@ In this guide, you will:
    :::image type="content" source="media\onelake-onecopy-quickstart\new-notebook-quickstart.png" alt-text="Screenshot of creating new notebook in Fabric." lightbox="media\onelake-onecopy-quickstart\new-notebook-quickstart.png":::
 
 1. Using the Fabric notebook, convert the CSV files to delta format. The following code snippet reads data from user created directory /Files/dimension_city and converts it to a delta table dim_city.
+   - Please note as for tables inside of Lakehouse: they should be created inside of the Table folder, only, and not using intermediate folders in between.
```

Comment:
I don't think the phrasing is very clear. Can you get someone from the lakehouse team like Ted to provide the right description please? He can also point you to the right location in the lakehouse docs to add it too.

Reply:
@tedvilutis as @eloldag mentioned can you review if I should rephrase or if you prefer to rephrase. 
This is related the email discussion:
RE: tables created on Databricks and transfer to Fabric
and the workitem: [https://msdata.visualstudio.com/DefaultCollection/Azure SQL Data Warehouse/_workitems/edit/2526429](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmsdata.visualstudio.com%2FDefaultCollection%2FAzure%2520SQL%2520Data%2520Warehouse%2F_workitems%2Fedit%2F2526429&data=05%7C01%7Clilem%40microsoft.com%7Cabea353c98394f2f270108db822b1dac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638246893953885074%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eXF5VTTvzyv2iS%2FpUrF1CzWmsm6ha%2Fgp4xJpqzzSgkE%3D&reserved=0)

Reply:
To help close on this PR, I came up with a wording that I think would be fine.  "Note: Load or shortcut Delta-Parquet data directly under the Table section of the lakehouse.   Do not nest your tables under additional subfolders in the Tables section because the lakehouse will not automatically recognize it as a table and label it as Unidentified."